docs.html

```diff
@@ -30,48 +30,84 @@
   <body>
     <div class="doc-page-layout">
       <!-- Navbar -->
-      <header class="navbar">
-        <nav class="container-xxl">
-          <a class="nav-brand" href="#">
-            <span class="nav-brand__maintxt">K</span>
-            <span class="nav-brand__sectxt">ube UI</span>
-          </a>
-          <button class="navbar-toggler" id="btn-toggle">
-            <span class="material-icons"> menu </span>
-          </button>
-          <div class="navbar-collapse collapse" id="nav-toggle">
-            <ul class="navbar-nav">
+      <header class="navbar navbar-home w-full"> 
+        <nav class="navbar-wrapper">
+          <div class="navbar-left-aligned">
+            <!-- <svg
+              class="sidebar-toggler"
+              id="sidebar-toggler"
+              viewBox="0 0 100 40"
+              width="40"
+              height="40"
+            >
+              <rect width="60" height="10"></rect>
+              <rect y="20" width="60" height="10"></rect>
+              <rect y="40" width="60" height="10"></rect>
+            </svg> -->
```

Comment:
when using SVG prefer storing it in a separate file in assets and using an IMG tag. unless you are manipulating the value in the svg with js